openfl/display/BitmapData.hx

```diff
@@ -514,6 +531,16 @@ class BitmapData implements IBitmapDrawable {
 		
 		surface.flush ();
 		
+		var renderer = @:privateAccess Lib.current.stage.__renderer;
+		var renderSession = @:privateAccess renderer.renderSession;
+		__drawGL (renderSession, this);
+		
+		if (Std.is(source, DisplayObject) && cast(source, DisplayObject).filters != null) {
+			
+			@:privateAccess BitmapFilter.__applyFilters (cast(source, DisplayObject).filters, renderSession, this, this, null, null);
```

Comment:
Yes, but this makes no sense, sorry. Just got to home, was quick in decisions :)